projects/kubernetes/descheduler.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    name: descheduler
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    name: descheduler-binary
+  spec:
+    failedBuildsHistoryLimit: 2
+    output:
+      to:
+        kind: ImageStreamTag
+        name: descheduler:binaries
+    source:
+      dockerfile: |
+        FROM scratch
+        COPY . /go/src/github.com/kubernetes-incubator/descheduler
+        RUN cd /go/src/github.com/kubernetes-incubator/descheduler && \
+            make
+      git:
+        uri: https://github.com/openshift/descheduler.git
+      type: Git
+    strategy:
+      dockerStrategy:
+        from:
+          kind: DockerImage
+          name: openshift/origin-release:golang-1.9
+        imageOptimizationPolicy: SkipLayers
+    successfulBuildsHistoryLimit: 1
+    triggers:
+    - type: ConfigChange
+    - type: GitHub
+      github:
+        secretReference:
+          name: github-deploymentconfig-trigger
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    name: descheduler
+  spec:
+    failedBuildsHistoryLimit: 2
+    output:
+      pushSecret:
+        name: registry-push
+      to:
+        kind: DockerImage
+        name: docker.io/openshift/descheduler:v1.2.4
```

Comment:
wonder why is the version here v1.2.4?

Reply:
fixed in https://github.com/openshift/release/pull/775